pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Create AsyncAPI Document for applications interacting with Kafka
```

Comment:
this is a specific tutorial that describes application that is consumer of kafka messages, as it has receive action

```suggestion
title: Create AsyncAPI Document for applications consuming from Kafka
```

wdyt? just a suggestion, can be generic as it is if you think it is better